ui/src/components/accountBids.js

```diff
@@ -0,0 +1,117 @@
+import React, { useContext, useEffect, useState } from 'react'
+import { Link } from 'react-router-dom';
+import { getBiddedItems } from '../apiConsumer/itemFetchConsumer';
+import AuthContext from '../context';
+
+function AccountBids(props){
+    const [bids,setBids]=useState([]);
+    
+    const context=useContext(AuthContext);
+
+    useEffect(()=>{
+        const token=context.jwt;
+
+        getBiddedItems(token,(success,data)=>{
+            if(success)
+                setBids(data);
+            else
+                setBids([]);
+        })
+    },[])
+
+    if(bids.length)
+        return(
+            <BiddedItems items={bids} />
+        )
+    else
+        return(
+            <div className="notSellerContainer">
+                <div className="notSellerTitle">No bids found</div>
+                <img className="notSellerImg" src="images/bid_purple.svg"></img>
+                <Link className="notSellerButton" to="/shop">Go to shop</Link>
+            </div>
+        )
+}
+
+function BiddedItems(props){
+    const [items,setItems]=useState([]);
+
+    useEffect(()=>{
+        setItems(props.items);
+    },[props.items]);
+
+    return(
+        <table className="accountSellerTable">
+            <tr className="accountSellerTableMedium">
+                <th className="accountSellerTable">Item</th>
```

Comment:
is this call here a mistake, this is table header, not table

Reply:
use `.accountSellerTable tr` instead of `tr.accountSellerTable`

Reply:
Not working?

Reply:
.accountSellerTable th *